main.go

```diff
@@ -2,31 +2,50 @@ package main
 
 import (
 	"fmt"
-	"log"
-	"os"
-	"runtime/pprof"
-
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
 	"github.com/uyuni-project/inter-server-sync/cli"
 	"github.com/uyuni-project/inter-server-sync/entityDumper"
 	"github.com/uyuni-project/inter-server-sync/schemareader"
+	"os"
+	"runtime/pprof"
 )
 
-func init() { log.SetFlags(log.Lshortfile | log.LstdFlags) }
+func loginit() {
+	Logfile := "/tmp/uyuni_iss_log.json"
+	lf, err := os.OpenFile(Logfile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	if os.IsNotExist(err) {
+		f, err := os.Create(Logfile)
+		if err != nil {
+			log.Error().Msg("Unable to create logfile")
+		}
+		lf = f
+	}
+	multi := zerolog.MultiLevelWriter(lf, os.Stdout)
+	log.Logger = zerolog.New(multi).With().Timestamp().Logger()
+
+}
 
 func main() {
+	loginit()
 	parsedArgs, err := cli.CliArgs(os.Args)
 	if err != nil {
+		log.Info().Msg("Not enough arguments")
```

Comment:
Since we are existing with an error code, should this be an `info` level or `error` level?